### DISCUSSION TITLE
Using alternative database server?

### DISCUSSION THREAD

**Discussion by oopiicaa** [hassio-addons/addon-traccar#232] [Q&A] [answered]
Is it possible to use this add-on with some other database server (MySQL for an instance)?
I found [this ](https://github.com/hassio-addons/addon-traccar/issues/5#issuecomment-505150857) comment from @frenck:

> How to use MariaDB/MySQL with this add-on has now been documented in the add-on documentation.
> Also, a recommendation to use it has been added to the installation steps.

I can't find anything in documentation. Was this removed? If not, should these steps/this kind of config work: https://www.traccar.org/mysql/? Currently I have Traccar running as a Windows app, connected to MySQL, but would like to migrate the app itself into Home Assistant (and keep current MySQL db).

**Comment by frenck** [comment, marked as answer]
> Is it possible to use this add-on with some other database server (MySQL for an instance)?

Sure, the configuration of traccar is exposed, feel free to modify it to your liking.

../Frenck
